Sources/ApolloSQLite/SQLiteNormalizedCache.swift

```diff
@@ -95,11 +96,30 @@ public final class SQLiteNormalizedCache {
     return try self.db.prepare(query).map { try parse(row: $0) }
   }
 
-  private func clearRecords() throws {
-    try self.db.run(records.delete())
-    if self.shouldVacuumOnClear {
-      try self.db.prepare("VACUUM;").run()
+  private func clearRecords(accordingTo policy: CacheClearingPolicy) throws {
```

Comment:
Optional: I think `with` instead of `accordingTo` would be more in line with other stuff in the lib already, but definitely not a hill i want to die on.

Reply:
Either reads fine, but `with` is shorter so I can make the quick update.

In my work's codebase we use `with` a lot that is overloaded so I have a slight aversion to that argument label :sweat_smile:

Reply:
Yeah we definitely use it a lot as well, but I think if you're not just going to say `policy` for both the internal and external param names, `with` is probably closer to how we want it to read at the call site